src/three-components/ARRenderer.js

```diff
@@ -198,7 +211,7 @@ export class ARRenderer extends EventDispatcher {
       this[$presentedScene].scaleModelToFitRoom();
     }
 
-    this[$frameOfReference] = null;
+    this[$refSpace] = null;
```

Comment:
Does changing `frameOfReference` to `refSpace` imply a corresponding change in the IDL?

Reply:
Yeah, this is an XRReferenceSpace rather than an XRFrameOfReference